kevm-pyk/src/tests/integration/test_prove.py

```diff
@@ -340,7 +339,7 @@ def test_prove_rules(
         tmp_path,
         caplog,
         no_use_booster,
-        force_sequential,
+        True,
```

Comment:
```suggestion
        force_sequential=True,
```

(Or whatever the name of that parameter is.)

Reply:
The other named parameters are left unnamed, should we name them all?